lib/libutils/isoc/include/stdio.h

```diff
@@ -12,6 +12,8 @@ typedef struct _FILE FILE;
 
 int printf(const char *fmt, ...)
                     __attribute__ ((__format__ (__printf__, 1, 2)));
+int sprintf(char *str, const char *fmt, ...)
```

Comment:
Comment for "libutils: add sprintf()"

This function is dangerous and should not be used. I suppose that some C++ thing needs it to be present but hopefully optimized away before use.
How about adding a warning or deprecated attribute that would show if it's not optimized away?

Reply:
It is used by `libstdc++`:
```
$ nm $(aarch64-linux-gnu-g++ -print-file-name=libstdc++.a) | grep sprintf
                 U sprintf
                 U sprintf
```

I'm fine with adding a `deprecated` attribute, or I can even remove the prototype from the header file and move it to the implementation file (`sprintf.c`) with a comment.

Reply:
I have added `__attribute__((deprecated))`.